### ISSUE TITLE
Change favicon on unread messages

### ISSUE THREAD

**Issue by SimenB** [gitterHQ/gitter#639]
Hey!

Really loving gitter, great app!

I always have a pinned tab with gitter, but this makes it hard to see when there's undread.
![screenshot](https://cloud.githubusercontent.com/assets/1404810/6913015/44e27602-d77a-11e4-8e27-65522905c19e.png)
Could you do something with the favicon when there's unread messages?  As of now, Chrome just makes it into a soft glow, I'd prefer the numbers you have now, or something like that.

**Comment by Laumania** [comment]
Yeah would be great, just like my gmail does (think I have an "app" turned on on my Gmail or something).
![gittericon](https://cloud.githubusercontent.com/assets/1378458/6918825/efe0a8da-d7b1-11e4-8b51-09b4d83129d8.png)

**Comment by quintesse** [comment]
Yes, this needs to be much more obvious, currently they change the default grey stripes to light green, but that's almost impossible to see. Maybe turn the entire icon green:

![image](https://cloud.githubusercontent.com/assets/778793/7090893/535607d8-dfa6-11e4-9ec9-2b2a15e399a8.png)

Also, as an additional feature: turn it a different color when there are messages mentioning **you**, eg:

![image](https://cloud.githubusercontent.com/assets/778793/7090940/9302edd8-dfa6-11e4-936d-02049de4ac02.png)

**Comment by MadLittleMods** [comment]
:+1: For a more noticeable difference.

Current: ![](http://i.imgur.com/d0JyrAy.gif)

**Comment by baev** [comment]
:+1:

**Comment by eroshenkoam** [comment]
:+1:

**Comment by vania-pooh** [comment]
:+1:

**Comment by smecsia** [comment]
:+1:
